src/utils.js

```diff
@@ -91,11 +91,21 @@ export const assert = function<T>(value: ?T): T {
     return value;
 };
 
+/**
+ * Return the protocol of a URL, or "_relative" if the URL does not specify a
+ * protocol (and thus is relative).
+ */
+export const urlToProtocol = function(url: string): string {
```

Comment:
`protocolFromUrl` or `getProtocol(FromUrl)`

Reply:
I'm going to go with `protocolFromUrl` since we use `fooFromBar` in a number of places already.